packages/editor/src/components/post-visibility/style.scss

```diff
@@ -0,0 +1,17 @@
+.edit-post-post-visibility__dialog {
+	.editor-post-visibility__dialog-legend {
```

Comment:
One of the advantage of the BEM-style class naming is that it avoids needing contextual nesting, since context is already built-in to the descendent class name.

In practical terms: The `.edit-post-post-visibility__dialog` nesting is unnecessary.